$MirOS: ports/lang/llvm/patches/patch-tools_lto_LTOCodeGenerator_cpp,v 1.1 2008/11/10 01:01:32 tg Exp $

	1) Support branding, especially via CPPFLAGS
	2) Fix whitespace mistake

--- tools/lto/LTOCodeGenerator.cpp.orig	Thu Jan  1 01:00:00 1970
+++ tools/lto/LTOCodeGenerator.cpp	Fri Dec  4 20:45:00 2009
@@ -59,11 +59,14 @@ using namespace llvm;
 static cl::opt<bool> DisableInline("disable-inlining",
   cl::desc("Do not run the inliner pass"));
 
+#ifdef LLVM_VERSION_INFO_SVN
+#define LLVM_VERSION_INFO " SVN " LLVM_VERSION_INFO_SVN
+#endif
 
 const char* LTOCodeGenerator::getVersionString()
 {
 #ifdef LLVM_VERSION_INFO
-    return PACKAGE_NAME " version " PACKAGE_VERSION ", " LLVM_VERSION_INFO;
+    return PACKAGE_NAME " version " PACKAGE_VERSION "," LLVM_VERSION_INFO;
 #else
     return PACKAGE_NAME " version " PACKAGE_VERSION;
 #endif
